sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/QuerySamples.cs

```diff
@@ -0,0 +1,235 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+using static Azure.IoT.TimeSeriesInsights.Samples.SampleLogger;
+
+namespace Azure.IoT.TimeSeriesInsights.Samples
+{
+    internal class QuerySamples
+    {
+        /// <summary>
+        /// This sample demonstrates querying for raw events and aggregate series data from a Time Series Insights environment.
+        /// </summary>
+        public async Task RunSamplesAsync(TimeSeriesInsightsClient client, DeviceClient deviceClient)
+        {
+            // Figure out what keys make up the Time Series Id
+            TimeSeriesModelSettings modelSettings = await client.ModelSettings.GetAsync().ConfigureAwait(false);
```

Comment:
Since this is used in 2 samples now, can think of moving to program.cs for reuse purpose.

Reply:
Good idea. Rather instead of moving to program.cs (to avoid circular dependency between both files), I'll probably create a new file called TimeSeriesIdHelper.cs. And add that logic there.